cmd/run.go

```diff
@@ -38,6 +43,13 @@ func Run(configFile string) (out Stoppable, err error) {
 		return nil, fmt.Errorf("Error while loading config file: %v", err)
 	}
 
+	if c.PprofPort > 0 {
+		go func() {
+			log.Printf("starting pprof listener on port %d", c.PprofPort)
+			http.ListenAndServe(fmt.Sprintf(":%d", c.PprofPort), nil)
```

Comment:
It might be worth printing out the error message returned by `ListenAndServer`. There may be another application already listening on `PprofPort`

Reply:
Good point. Doing that now.